google/cloud/storage/benchmarks/benchmark_utils.cc

```diff
@@ -118,6 +119,78 @@ std::chrono::seconds ParseDuration(std::string const& val) {
   return std::chrono::seconds(s);
 }
 
+bool ParseBoolean(std::string const& val, bool default_value) {
+  if (val.empty()) {
+    return default_value;
+  }
+  auto lower = val;
+  std::transform(lower.begin(), lower.end(), lower.begin(),
+                 [](char x) { return static_cast<char>(std::tolower(x)); });
+  if (lower == "true") {
+    return true;
+  } else if (lower == "false") {
+    return false;
+  }
+  throw std::runtime_error("Cannot parse " + val + " as a boolean");
+}
+
+std::string Basename(std::string const& path) {
+  // With C++17 we would use std::filesytem::path, until then do the poor's
+  // person version.
+#if _WIN32
+  return path.substr(path.find_last_of('\\') + 1);
```

Comment:
Windows should probably `find_last_of("\\/")`.

Reply:
Done.